src/sqlfluff/rules/L016.py

```diff
@@ -263,8 +280,9 @@ def generate_fixes_to_coerce(
         elif indent_impulse:  # pragma: no cover
             role = "breakpoint"
         else:
-            raise ValueError("Is this possible?")  # pragma: no cover
-
+            # This can happen, e.g. with a long template line. Treat it as
```

Comment:
This is the actual fix. When this happens, we assume it's unfixable. Presumably, the line is too long in the **source** space because of Jinja templating. As designed, this rule makes changes in SQL space, i.e. it's not going to fix template code.